index.html

```diff
@@ -30,71 +30,264 @@
+                <li class="language-list-item works-grid-one-project-language-list">Ruby</li>
+              </ul>
+            </div>
+            <button type="button" class="grid-project-button button">See Project</button>
+          </div>
+          <div class="works-grid-one-project-container fifth-grid">
+            <div class="grid-project1 works-grid-project">
+              <h3 class="works-grid-one-project-header">Profesional Art Printing Data</h3>
+              <p class="works-grid-one-project-description">
+                Lorem ipsum dolor sit, amet consectetur adipisicing elit. 
+                Accusamus recusandae illum velit amet consectetur omnis aperiam aut. 
+                Incidunt quaerat 
+              </p>
+    
+              <ul class="language-list-body">
+                <li class="language-list-item works-grid-one-project-language-list">HTML</li>
+                <li class="language-list-item works-grid-one-project-language-list">Bootstrap</li>
+                <li class="language-list-item works-grid-one-project-language-list">Ruby</li>
+              </ul>
+            </div>
+            <button type="button" class="grid-project-button button">See Project</button>
+          </div>
+          <div class="works-grid-one-project-container sixth-grid">
+            <div class="grid-project1 works-grid-project">
+              <h3 class="works-grid-one-project-header">Profesional Art Printing Data</h3>
+              <p class="works-grid-one-project-description">
+                Lorem ipsum dolor sit, amet consectetur adipisicing elit. 
+                Accusamus recusandae illum velit amet consectetur omnis aperiam aut. 
+                Incidunt quaerat 
+              </p>
+    
+              <ul class="language-list-body">
+                <li class="language-list-item works-grid-one-project-language-list">HTML</li>
+                <li class="language-list-item works-grid-one-project-language-list">Bootstrap</li>
+                <li class="language-list-item works-grid-one-project-language-list">Ruby</li>
+              </ul>
+            </div>
+            <button type="button" class="grid-project-button button">See Project</button>
+          </div>
+        </div>
+    </section>
+    <!--end-->
+    <!--About me-->
+    <section class="about">
+    <div class="about-section">
+        <div class="about-sub-section">
+        <h1 class="about-header header">About me</h1>
+        <p class="about-description">
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hestiate to contact me.
+        </p>
+        <button type="button" class="about-button button">Get My Resume</button>
+        </div>
+        <div class="about-illustration">
+        <img src="./ressources/illustration-about-me-mobile.png" alt="img">
```

Comment:
this illustration is just for design purposes so please try to use it as a background image with CSS